SPECS/glusterfs/glusterfs.spec

```diff
@@ -574,55 +718,56 @@ sed -i -e 's/--quiet//' configure.ac
         %{?_without_fusermount} \
         %{?_without_georeplication} \
         %{?_without_ocf} \
-        %{?_without_rdma} \
         %{?_without_server} \
         %{?_without_syslog} \
         %{?_with_ipv6default} \
-        %{?_without_libtirpc}
-
-# fix hardening and remove rpath in shlibs
+        %{?_without_linux_io_uring} \
+        %{?_without_libtirpc} \
+        %{?_without_tcmalloc}
 
+## fix hardening and remove rpath in shlibs
 sed -i 's| \\\$compiler_flags |&\\\$LDFLAGS |' libtool
-
 sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|' libtool
 sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|' libtool
 
-gcc -v
 %make_build
 
 %check
 make check
 
 %install
-%make_install
-
+make install DESTDIR=%{buildroot}
 %if ( 0%{!?_without_server:1} )
+%if ( 0%{_for_fedora_koji_builds} )
```

Comment:
I'd remove all references to `_for_fedora_koji_builds`. Right now I think we're not erroring-out, because these `%if` statements are never analyzed by RPM due to the earlier check for `_without_server` or other macros, which evaluate to a false statement. But these Fedora macros will never be present in our system and with the way they are used now, they are prone to unnecessarily generate failures. I'd just assume the macro is not defined and update the spec accordingly.